lib/ethereum/cw_ethereum.dart

```diff
@@ -30,14 +30,23 @@ class CWEthereum extends Ethereum {
   }) =>
       EVMChainRestoreWalletFromPrivateKey(name: name, password: password, privateKey: privateKey);
 
+  @override
+  WalletCredentials createEthereumHardwareWalletCredentials({
+    required String name,
+    required HardwareAccountData hwAccountData,
+    WalletInfo? walletInfo,
+  }) =>
+      EVMChainRestoreWalletFromHardware(
+          name: name, hwAccountData: hwAccountData, walletInfo: walletInfo);
+
   @override
   String getAddress(WalletBase wallet) => (wallet as EthereumWallet).walletAddresses.address;
 
   @override
   String getPrivateKey(WalletBase wallet) {
     final privateKeyHolder = (wallet as EthereumWallet).evmChainPrivateKey;
-    String stringKey = bytesToHex(privateKeyHolder.privateKey);
-    return stringKey;
+    if (privateKeyHolder is EthPrivateKey) return bytesToHex(privateKeyHolder.privateKey);
+    return "";
```

Comment:
this will make the WalletConnect feature fail, and I don't see any handlers for this flow